jekyll/_cci2/Pipelines.md

```diff
@@ -0,0 +1,42 @@
+---
```

Comment:
sorry, we can't handle renames, so this filename should be changed back to build-processing.md

Reply:
Renamed doc back to original "Build-Processing.md"